### PR TITLE
Added labs to items_metadata.yaml file

### PR DESCRIPTION
**Pull request by DavidJVitale**
We had a lot of 404s on the dev site since the labs pointed to the old notebooks.esri.com URLs. This adds metadata to the items_metadata.yaml file and uploads the items as notebooks to an AGOL org.

Coordinated with an upcoming dev site release

### CODE REVIEW

**Comment by AtmaMani** [comment]
@mohi9282 can you review this?

File: samples/04_gis_analysts_data_scientists/covid19_part2_timeseries_analysis.ipynb [hunk 1]
```diff
@@ -3680,7 +3680,7 @@
     "\n",
```
Comment (mohi9282) [inline samples/04_gis_analysts_data_scientists/covid19_part2_timeseries_analysis.ipynb:3680]: <p>Hi Dave - The link to StoryMap in [here] looks good but the link to 'this notebook' looks weird, it is showing me "https://app.reviewnb.com/Esri/arcgis-python-api/pull/819/files/covid19_part3_predictive_analysis.ipynb". </p><p> </p><p>Is this how a link to sample notebook would look like in ReviewNB and if not could you please fix the link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Comment (DavidJVitale) [reply samples/04_gis_analysts_data_scientists/covid19_part2_timeseries_analysis.ipynb:3680]: Hi Mohit that is expected. ReviewNB isn't aware of other notebooks in the local directory, so you can ignore any invalid links when reviewing in reviewNB. Invalid links when you run the notebook locally are what we are concerned about

**Review by mohi9282 — CHANGES_REQUESTED** [review]
Hi @DavidJVitale - In items_metadata.yaml file "display_web_map" code block is duplicated line 660-667 and 708-715. Could you please remove one.

**Review by mohi9282 — APPROVED** [review]
Looks good

**Pull request merged**